go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/jamiealquiza/bicache
```

Comment:
I'm happy to move the go modularization commit into its own PR as well, if you'd prefer -- let me know